hoot-core/src/main/cpp/hoot/core/algorithms/alpha-shape/AlphaShape.cpp

```diff
@@ -345,7 +345,7 @@ GeometryPtr AlphaShape::toGeometry()
     for (EdgeIterator it = _pDelauneyTriangles->getEdgeIterator(); it != _pDelauneyTriangles->getEdgeEnd(); ++it)
     {
       //  Multiply by scale and convert it to a whole number for deduplication in the set
-      long length = (*it).getLength() * scale;
+      double length = (*it).getLength() * scale;
```

Comment:
This must be a double, see the comment above.  The proper way to fix this might be one of the following:
```
long length = (long)((*it).getLength() * scale);
//  or
long length = std::static_cast<int)((*it).getLength() * scale);
```